content/manuals/engine/release-notes/28.md

```diff
@@ -22,6 +23,77 @@ For more information about:
 - Deprecated and removed features, see [Deprecated Engine Features](../deprecated.md).
 - Changes to the Engine API, see [Engine API version history](/reference/api/engine/version-history.md).
 
+## 28.1.0
+
+{{< release-date date="2025-04-17" >}}
+
+For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:
+
+- [docker/cli, 28.1.0 milestone](https://github.com/docker/cli/issues?q=is%3Aclosed+milestone%3A28.1.0)
+- [moby/moby, 28.1.0 milestone](https://github.com/moby/moby/issues?q=is%3Aclosed+milestone%3A28.1.0)
+
+### New
+
+- Add `docker bake` sub-command as alias for `docker buildx bake`. [docker/cli#5947](https://github.com/docker/cli/pull/5947)
+- Add a new flag `--use-api-socket` to enable access to Docker socket from inside a container. [docker/cli#5858](https://github.com/docker/cli/pull/5858)
```

Comment:
We probably need `experimental` here; let me update the description.

I updated it to;

```
- Experimental: add a new `--use-api-socket` flag on `docker run` and `docker create` to enable access to Docker socket from inside a container and to share credentials from the host with the container.
```

(better suggestions welcome!)